week1_KimChaeHyun/src/components/header/Header.jsx

```diff
@@ -0,0 +1,39 @@
+import React , { useState } from 'react';
+import styles from './Header.module.css';
+import  { Logo, Notification, Search, Toggle }  from '@assets/svgs';
+
+const Header = () => {
+  const [isDropdownOpen, setIsDropdownOpen] = useState(false);
+
+  const handleToggleClick = () => {
+    console.log("토글오픈");
+    setIsDropdownOpen(!isDropdownOpen);
```

Comment:
```suggestion
    setIsDropdownOpen((prev) => !prev);
```
css 관련은 아닌데 이게 좀 더 안전할듯용?